sentry-contexts/src/utils.rs

```diff
@@ -71,6 +105,14 @@ mod model_support {
     pub fn get_family() -> Option<String> {
         None
     }
+
+    pub fn get_macos_version() -> Option<String> {
+        None
+    }
+
+    pub fn get_macos_build() -> Option<String> {
+        None
+    }
```

Comment:
I don’t think these are needed, I’m surprised this is not flagged as unused code even.

Reply:
Ah, it is indeed flagged as unused in CI ;-)